luigi/task.py

```diff
@@ -49,6 +50,30 @@
 _SAME_AS_PYTHON_MODULE = '_same_as_python_module'
 
 
+if sys.version_info < (3, 9):
+    try:
+        from typing_extensions import Annotated, get_args, get_origin, get_type_hints
+    except ImportError:
+        Annotated = get_args = get_origin = get_type_hints = None
+else:
+    from typing import Annotated, get_args, get_origin, get_type_hints
+
+
+def _get_pep539_annotations(tsk_cls):
```

Comment:
I'm not a fan of having the pep593 in the method name and also in the case there's a typo:

```suggestion
def _get_pep593_annotations(tsk_cls):
```